### PR TITLE
Animate bubble dialog

### PR DESCRIPTION
**Pull request by balderdash**
Animate a few parts of the new dialog:
* Odometer-style block counter that spins up to your block count
* Big bubble at the top fades in and your block count "badge" pops up after your blocks finish counting
* Achievements that you got fade in one at a time

dialog2

Based on Mark's motion mock here: https://share.proto.io/1XIMDP/

### CODE REVIEW

File: apps/src/templates/Odometer.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    height: 36,
+    width: 21,
+    overflowX: 'hidden',
+  },
+};
+
+export default class Odometer extends Component {
+  constructor() {
+    super();
+  }
```
Comment (joshlory) [inline apps/src/templates/Odometer.js]: Not needed?

File: apps/src/templates/FinishDialog.jsx [hunk 2]
```diff
@@ -146,31 +161,74 @@ export class UnconnectedFinishDialog extends Component {
...
+      description = msg.perfectDescription();
+    } else {
+      description = msg.tooManyBlocksDescription();
     }
-    if (this.props.blocksUsed > this.props.blockLimit) {
-      return msg.tooManyBlocksDescription();
+
+    if (this.state.blocksCounted) {
+    } else {
     }
```
Comment (joshlory) [inline apps/src/templates/FinishDialog.jsx]: Not used?

Comment (balderdash) [reply apps/src/templates/FinishDialog.jsx]: Whoops, thanks.

File: apps/src/templates/Odometer.js (hunk 1 above)
Comment (balderdash) [reply apps/src/templates/Odometer.js]: Ditto.

**Pull request merged**